.github/actions/build-json-module-and-redis/action.yml

```diff
@@ -7,6 +7,8 @@ inputs:
     description: 'Redis version to build'
     required: true
     default: '7.2.1'
+  san:
```

Comment:
It is a little difficult to guess what "san" may stand for. Can we make it longer yet fully self-explanatory?